project/src/js/socket.js

```diff
@@ -20,13 +20,24 @@ let redrawOnDisconnectedTimeoutID;
 let proxyFailTimeoutID;
 const proxyFailMsg = 'The connection to lichess server has failed. If the problem is persistent this may be caused by proxy or network issues. In that case, we\'re sorry: lichess online features such as games, connected friends or challenges won\'t work.';
 
+const userPing = m.prop();
+const serverMoveTime = m.prop();
+
 const defaultHandlers = {
   following_onlines: handleFollowingOnline,
   following_enters: name => autoredraw(() => friendsApi.add(name)),
   following_leaves: name => autoredraw(() => friendsApi.remove(name)),
   challenges: data => {
     challengesApi.set(data);
     m.redraw();
+  },
+  mlat: mlat => {
```

Comment:
Great point. I will fix.

On Tue, Aug 16, 2016 at 7:55 AM, Vincent Velociter <notifications@github.com
> wrote:

> In project/src/js/socket.js
> <https://github.com/veloce/lichobile/pull/392#discussion_r74920161>:
>
> >  const defaultHandlers = {
> >    following_onlines: handleFollowingOnline,
> >    following_enters: name => autoredraw(() => friendsApi.add(name)),
> >    following_leaves: name => autoredraw(() => friendsApi.remove(name)),
> >    challenges: data => {
> >      challengesApi.set(data);
> >      m.redraw();
> > +  },
> > +  mlat: mlat => {
>
> That can't be done like this. Since it will trigger a redraw on each ping
> event, that's a lot of useless redraws. The redraws should happend only
> when menu is open, and not all the time.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/veloce/lichobile/pull/392/files/16d48ab4f4a57b7af91c045128bbdc482b54cd02#r74920161>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6RdqwxfPtTAEJX4rrf3DbLXwmxtD-2ks5qgaUcgaJpZM4JcRdr>
> .
>